README.md

```diff
@@ -129,6 +129,19 @@ the steps accordingly.
 
 That's it!  The service should now be running and working.
 
+## Upgrade
+
+Upgrade the service to the latest git version with these stepos:
+
+```
+cd /opt/ff-node-monitor/src/
+sudo systemctl stop ff-node-monitor
+git pull
+sudo -u ff-node-monitor /opt/ff-node-monitor/.cargo/bin/rustup default $(cat rust-version)
+sudo -u ff-node-monitor /opt/ff-node-monitor/.cargo/bin/cargo build --release
+sudo systemctl start ff-node-monitor
+```
+
 ### Integration
```

Comment:
This should probably also become a 2nd-level header then?

Reply:
done